Packs/FiltersAndTransformers/Scripts/AfterRelativeDate/AfterRelativeDate.yml

```
@@ -1,8 +1,8 @@
 args:
-- description: Date value to check - Can be any non-timezone aware time format. ex. "2020-01-01T00:00:00"
+- description: Date value to check - Can be any non-timezone aware time format. ex. "2020-01-01T00:00:00".
```

```suggestion
- description: Date value to check - Can be any time format. ex. "2020-01-01T00:00:00".
```
update the readme as well